frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/index.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { 
+  Form, 
+  FormGroup, 
+  TextInput,
+  Button,
+  ValidatedOptions,
+ } from '@patternfly/react-core';
+import { EyeSlashIcon, EyeIcon } from '@patternfly/react-icons';
+import {
+  SecretKind,
+  apiVersionForModel,
+} from '@console/internal/module/k8s';
+import {
+  SecretModel
+} from '@console/internal/models';
+import { isValidUrl } from '@console/shared';
+import { CreatePayload, ExternalComponentProps, CanGoToNextStep } from '../types';
+import { FlashsystemState, IBMFlashsystemKind } from './type';
+import { IBMFlashsystemModel } from './models';
+
+export const FlashsystemConnectionDetails: React.FC<ExternalComponentProps<FlashsystemState>> = ({ 
+  setFormState,
+  formState,
+}) => {
+  const { t } = useTranslation();
+  const [reveal, setReveal] = React.useState(false);
+  const [endpointValid, setEndpointValid] = React.useState(ValidatedOptions.default);
+  const onChange = (value: string) => {
+    setFormState('endpoint', value);
+    if (value){
+      if (isValidUrl(value)){
+        setEndpointValid(ValidatedOptions.success);
+      } else {
+        setEndpointValid(ValidatedOptions.error);
+      }
+    };
+  };
```

Comment:
nit : add some space
```suggestion

  const onChange = (value: string) => {
    setFormState('endpoint', value);
    if (value){
      if (isValidUrl(value)){
        setEndpointValid(ValidatedOptions.success);
      } else {
        setEndpointValid(ValidatedOptions.error);
      }
    };
  };
```